multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/executor/TimeoutAsyncJobExecutor.java

```diff
@@ -0,0 +1,13 @@
+package org.cloudfoundry.multiapps.controller.process.executor;
+
+import org.flowable.job.service.impl.asyncexecutor.DefaultAsyncJobExecutor;
+
+public class TimeoutAsyncJobExecutor extends DefaultAsyncJobExecutor {
+
+    @Override
+    protected void initAsyncJobExecutionThreadPool() {
+        TimeoutAsyncTaskExecutor taskExecutor = (TimeoutAsyncTaskExecutor) this.taskExecutor;
+        taskExecutor.start();
+        shutdownTaskExecutor = true;
```

Comment:
What is this variable used for? Seems confusing that we are telling the executor to shut down right after starting it.

Reply:
It comes from flowable and unfortunately, there is no setter for it. If you take a look at the `super` implementation, you will see it also.